### DISCUSSION TITLE
Unknown Kdf - Webvault Version 2022.12.0

### DISCUSSION THREAD

**Discussion by Cameo007** [dani-garcia/vaultwarden#3443] [Q&A] [answered]
I wanted to login to my web vault but after  entering my password I saw the error message "Unknown Kdf" so I couldn't login. The browser plugin, the desktop app (Windows) and the android app are working.

I can see these error messages in my browser's console:
```
Error: Unknown Kdf.
    at qo.<anonymous> (crypto.service.ts:422:13)
    at Generator.next (<anonymous>)
    at main.5f8690f5c03a207c390a.js:54:510166
    at new t (zone.js:1351:25)
    at So (main.5f8690f5c03a207c390a.js:54:509911)
    at qo.makeKey (main.5f8690f5c03a207c390a.js:54:517260)
    at useClass.<anonymous> (auth.service.ts:262:31)
    at Generator.next (<anonymous>)
    at r (main.5f8690f5c03a207c390a.js:54:403766)
    at t.invoke (zone.js:409:30)
```
and
```
Received API exception: Error: Unknown Kdf.
```

both from `consoleLog.service.ts:51`

**Comment by BlackDex** [comment]
Which version of Vaultwarden are you running, how did you installed it?
We need more info.

**Comment by BlackDex** [comment, marked as answer]
Ow, I see you have `web-vault` 2022.12.0. That is too old. Upgrade Vaultwarden to the latest version. That will solve your issues.

**Reply by Cameo007** [reply]
That's strange because I'm using the latest tag from Docker Hub and I had been using it the last day. I stopped the container and started it again and now it works with v2023.3.0.

**Reply by BlackDex** [reply]
Maybe the re-creation of the container didn't went well the first time.
No clue. But glad it is working now.

**Reply by Cameo007** [reply]
Thanks for your help!
